hs_core/migrations/0063_auto_20220601_2014.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2022-06-01 20:14
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+def remove_pending_citations_from_relations(apps, schema_editor):
```

Comment:
@hyi I guess we could do this with a periodic task instead, but I think it's better if we keep "DOI pending" messages out of the `Relation` from now on and remove it from the existing ones